sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/schemas/AwsSchemaProvider.java

```diff
@@ -75,9 +75,11 @@ public class AwsSchemaProvider extends GetterBasedSchemaProvider {
 
   @SuppressWarnings("rawtypes")
   @Override
-  public List<FieldValueGetter> fieldValueGetters(Class<?> clazz, Schema schema) {
+  public List<FieldValueGetter> fieldValueGetters(
```

Comment:
Noticed that we break backwards compatibility of public fields here but probably this is OK since these are expected be util classes intended only internal usage ?

Reply:
good point - this is the consequence of changing the methods' signatures in the `GetterBasedSchemaProvider` - it's not annotated with `@Internal` so maybe we should actually remain careful with that interface - maybe I'll add some default implementations of the backward compatible methods to that class which